java/api/pom.xml

```diff
@@ -43,4 +43,13 @@
   <version>1.7.25</version>
 </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+      </plugin>
+    </plugins>
+  </build>
```

Comment:
Is it possible to only set this in the root pom.xml file?

Reply:
Good catch, just fixed